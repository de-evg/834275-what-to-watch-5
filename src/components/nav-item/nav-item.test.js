import React from "react";
import renderer from "react-test-renderer";
import NavItem from "./nav-item";

const DEFAULT_TITLE = `Overview`;

it(`Should NavItem render correctly`, () => {
  const tree = renderer
    .create(<NavItem
      onNavItemChange={() => {}}
      isActive={Math.random < 0.5}
      title={DEFAULT_TITLE}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
