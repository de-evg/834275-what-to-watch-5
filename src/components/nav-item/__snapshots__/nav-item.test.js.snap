// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NavItem render correctly 1`] = `
<li
  className="movie-nav__item "
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Overview
  </a>
</li>
`;
